pytorch_blade/src/torch-mlir/lib/Dialect/TorchConversion/Transforms/DiscLowerOperatorOp.cpp

```diff
@@ -0,0 +1,160 @@
+// Copyright 2022 The BladeDISC Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "mlir/Dialect/Arithmetic/IR/Arithmetic.h"
+#include "mlir/Dialect/ControlFlow/IR/ControlFlowOps.h"
+#include "mlir/Dialect/Func/IR/FuncOps.h"
+#include "mlir/Dialect/Math/IR/Math.h"
+#include "mlir/Dialect/Tensor/IR/Tensor.h"
+#include "mlir/IR/Builders.h"
+#include "mlir/Transforms/DialectConversion.h"
+#include "stablehlo/dialect/ChloOps.h"
+#include "tensorflow/compiler/mlir/disc/IR/hlo_disc_ops.h"
+#include "torch-mlir/Conversion/MhloPasses.h"
+#include "torch-mlir/Conversion/TorchToMhlo/TorchToMhlo.h"
+#include "torch-mlir/Dialect/Torch/IR/TorchDialect.h"
+#include "torch-mlir/Dialect/Torch/IR/TorchOps.h"
+#include "torch-mlir/Dialect/Torch/IR/TorchTypes.h"
+#include "torch-mlir/Dialect/TorchConversion/Transforms/BackendTypeConversion.h"
+
+using namespace mlir;
+using namespace mlir::torch;
+using namespace mlir::torch::Torch;
+using namespace mlir::torch::TorchConversion;
+
+namespace {
+
+class ConvertOperatorOp : public OpConversionPattern<OperatorOp> {
+ public:
+  using OpConversionPattern<OperatorOp>::OpConversionPattern;
+  using OpAdaptor = typename OperatorOp::Adaptor;
+  LogicalResult matchAndRewrite(
+      OperatorOp op,
+      OpAdaptor adaptor,
+      ConversionPatternRewriter& rewriter) const override {
+    Location loc = op.getLoc();
+    auto name = op.name();
+    if ("torch_blade.fake_quant" == name) {
+      auto operands = adaptor.operands();
+      auto input = operands[0];
+      auto scale = operands[1];
+      auto zeroPoint = operands[2];
+
+#define I64_VAR_FROM_CONST_OPERAND(var, idx)                         \
+  int64_t var;                                                       \
```

Comment:
Please format the indent

Reply:
It's already formated by `clang-format-10`....

Reply:
TorchBlade will follow PyTorch's clang-format